pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1111,6 +1120,17 @@ public CompletableFuture<Optional<Topic>> getTopic(final TopicName topicName, bo
         }
     }
 
+    private CompletableFuture<Optional<TopicPolicies>> getTopicPoliciesBypassSystemTopic(@Nonnull TopicName topicName) {
+        Objects.requireNonNull(topicName);
+        if (pulsar.getConfig().isTopicLevelPoliciesEnabled()
+                && !NamespaceService.isSystemServiceNamespace(topicName.getNamespace())
```

Comment:
Do we need to add

&& pulsar.getConfig().isSystemTopicEnabled()?

Reply:
I don't think we should add this condition because `pulsar.getConfig().isTopicLevelPoliciesEnabled()` has a precondition to enable the system topic. Otherwise, the topic policy service will be disabled. 

Considering pluggable topic-level policies service, it shoudn't be bind to system topic. :)

Reply:
Now, we have 

```
            if (config.isTopicLevelPoliciesEnabled() && config.isSystemTopicEnabled()) {
                this.topicPoliciesService = new SystemTopicBasedTopicPoliciesService(this);
            }
```

But users still can enable the topic policies but disable the system topic. That doesn't make sense. But before we force require (not allow to start the broker) the system topic set to true while topic policies enabled, should we also check the system topic? After we applied the force precondition, then we can remove all of the system topic checks.

Reply:
Okay, I think you convinced me. From a design perspective, it will be more precise. But from a performance perspective, it's not worth spending more conditions to support a low-probability and no-impact judgment statement. (but it's okay because Pulsar already has too many no-impact things. :/ )

Reply:
fixed.